rgdc/rgdc/rgdc.py

```diff
@@ -206,19 +206,85 @@ def search(
         relates: Optional[str] = None,
         distance: Optional[Tuple[float, float]] = None,
         acquired: Optional[DATETIME_OR_STR_TUPLE] = None,
-        created: Optional[DATETIME_OR_STR_TUPLE] = None,
-        modified: Optional[DATETIME_OR_STR_TUPLE] = None,
-        datatype: Optional[SEARCH_DATATYPE_CHOICE] = None,
```

Comment:
If `datatype` won't be used anymore we should remove `SEARCH_DATATYPE_CHOICE` from `types.py`

Reply:
Done